index.html

```diff
@@ -0,0 +1,252 @@
+              We invite you to participate in the three-day conference, present
+              your research work, and contribute to transforming the way the
+              world engages with issues of gender and sexuality, publish with
+              well-known publishers while advancing your career.
+            </p>
+          </div>
+          <h3 class="header-2 color-dark-text mb-12 ff-2">
+            2021.25.06(FRI) ~ 27(SAT)
+          </h3>
+          <p class="body-2 color-dark-text">
+            @ Alexis Hotel Conference Center, Lilongwe, Malawi 
+          </p>
+        </div>
+      </article>
+      <section class="main-program bg-dark color-white">
+        <h2 class="header-3 text-center">Main Program</h2>
+        <div class="divider bg-primary bd-primary my-12"></div>
+        <ul class="program-list">
+          <li class="program-item">
+            <i class="fas fa-microphone"></i>
+            <p class="color-primary program-title">8 Keynote Addresses</p>
+            <p class="program-description">
+              Delivered by high level dignitaries
+            </p>
+          </li>
+          <li class="program-item">
+            <i class="fas fa-people-arrows"></i>
+            <p class="color-primary program-title">6 high level Panels</p>
+            <p class="program-description">
+              Featuring society leaders; gender experts and young women
+            </p>
+          </li>
+          <li class="program-item">
+            <i class="far fa-comment-dots"></i>
+            <p class="color-primary program-title">20+ #CovidVoices</p>
+            <p class="program-description">
+              Stories of how Covid19 has affected women and girls from across Africa
+            </p>
+          </li>
+          <li class="program-item">
+            <i class="fas fa-user-friends"></i>
+            <p class="color-primary program-title">Workshops</p>
+            <p class="program-description">
+              Partner organised Skills Building workshops
+            </p>
+          </li>
+          <li class="program-item">
+            <i class="far fa-flag"></i>
+            <p class="color-primary program-title">Side events</p>
+            <p class="program-description">
+              Partner organised Side events and Satellite events 
+            </p>
+          </li>
+          <li class="program-item">
+            <i class="fas fa-guitar"></i>
+            <p class="color-primary program-title">Concert</p>
+            <p class="program-description">
+              A Concert and various music, dance and poetry performances
+            </p>
+          </li>
+          <li class="program-item">
+            <i class="fas fa-trophy"></i>
+            <p class="color-primary program-title">Annual AGE Awards</p>
+            <p class="program-description">
+              A celebration of individuals and organisations that have championed gender equality.
+            </p>
+          </li>
+        </ul>
```

Comment:
- [ ] I think you should limit the number of articles in this section to five(5). Having more than five(5) as it is, will cause the other articles to break to the next page, which is not a good user experience for desktop screen sizes. Also, looking at the design, the articles are restricted to five(5).
- [ ] I also think you should use the `<article>` tag for each of the list items in this article, as it is more meaningful semantically.
- [ ] Lastly here, the line of text at the bottom of the articles is missing in your design. Kindly add it.

![original](https://user-images.githubusercontent.com/26856491/123656235-a47e5900-d827-11eb-8346-f6c3644eb3e1.png)
_**Original Design**_

Reply:
@teekaytech Using the `<article>` gives a linter warning due to the absence of a header element immediately after the opening tag.

Reply:
In that case, use it without linters errors, that's the priority. 🐱